packages/salesforcedx-vscode-core/src/commands/isvdebugging/bootstrapCmd.ts

```diff
@@ -386,39 +388,47 @@ export class IsvDebugBootstrapExecutor extends SfdxCommandletExecutor<{}> {
     );
 
     // 7a: unzip downloaded packages into temp location
-    try {
-      const packagesTempPath = path.join(projectMetadataTempPath, 'packages');
-      shell.mkdir('-p', packagesTempPath);
-      shell.mkdir('-p', projectInstalledPackagesPath);
-      const zip = new AdmZip(
-        path.join(projectMetadataTempPath, 'unpackaged.zip')
-      );
-      zip.extractAllTo(packagesTempPath, true);
-    } catch (error) {
-      console.error(error);
-      channelService.appendLine(
-        nls.localize('error_extracting_packages', error.toString())
-      );
-      notificationService.showErrorMessage(
-        nls.localize('error_extracting_packages', error.toString())
-      );
-      return;
-    }
+    // try {
+    //   const packagesTempPath = path.join(projectMetadataTempPath, 'packages');
+    //   shell.mkdir('-p', packagesTempPath); // .sfdx/tools/isvdebuggermdapitmp/packages
+    //   shell.mkdir('-p', projectInstalledPackagesPath); // .sfdx/tools/installed-packages
+    //   const zip = new AdmZip(
+    //     path.join(projectMetadataTempPath, 'unpackaged.zip')
+    //   );
+    //   zip.extractAllTo(packagesTempPath, true);
+    // } catch (error) {
+    //   console.error(error);
+    //   channelService.appendLine(
+    //     nls.localize('error_extracting_packages', error.toString())
+    //   );
+    //   notificationService.showErrorMessage(
+    //     nls.localize('error_extracting_packages', error.toString())
+    //   );
+    //   return;
+    // }
 
     // 7b: convert packages into final location
+    // const packagesTempPath = path.join(projectMetadataTempPath, 'packages');
+    // shell.cp('-r', packagesTempPath, projectInstalledPackagesPath);
     for (const packageInfo of packageInfos) {
       channelService.appendLine(
         nls.localize('isv_debug_bootstrap_processing_package', packageInfo.name)
       );
-      await this.executeCommand(
-        this.buildMetadataApiConvertPackageSourceCommand(packageInfo.name),
-        { cwd: projectPath },
-        cancellationTokenSource,
-        cancellationToken
-      );
+      // await this.executeCommand(
+      //   this.buildMetadataApiConvertPackageSourceCommand(packageInfo.name),
+      //   { cwd: projectPath },
+      //   cancellationTokenSource,
+      //   cancellationToken
+      // );
 
       // generate installed-package.json file
       try {
+        shell.cp('-r', path.join(
```

Comment:
This can be done using fs.renameSync()

Reply:
I figured out the cmd to retrieve packages directly in the destination folder, so we won't need to copy now.